### PR TITLE
Fixes #28340 - Fix Login Page styles

### PR DESCRIPTION
**Pull request by Ron-Lavi**
After a minor change in https://github.com/theforeman/foreman/pull/7035
the styles got broken.

In this PR I am removing a leftover css-class
and adjusting the existing login page scss file.

should be cherry-picked to 1.24

### CODE REVIEW

File: app/views/layouts/login.html.erb [hunk 1]
```diff
@@ -1,8 +1,6 @@
 <%= content_for(:content) do %>
   <%= notifications %>
-  <div class="login-page" >
```
Comment (tbrisker) [inline app/views/layouts/login.html.erb:3]: This class is also used in the external logout scss

**Comment by tbrisker** [comment]
also, [#24990](https://projects.theforeman.org/issues/24990) has already been released, please use a new issue.

File: app/views/layouts/login.html.erb (hunk 1 above)
Comment (Ron-Lavi) [reply app/views/layouts/login.html.erb:3]: good catch ! updated that file too.

**Comment by Ron-Lavi** [comment]
Thanks, updated issue: #28340

**Review by tbrisker — APPROVED** [review]

**Comment by tbrisker** [comment]
1.24 - 8e714bfa2

**Pull request merged**